packages/eslint/common/no-external-imports/index.js

```diff
@@ -0,0 +1,84 @@
+// require('./typings.d.ts');
+const path = require('path');
+const fs = require('fs');
+
+const DEPENDENCIES_VARIANTS = ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'];
+
+module.exports = (filenamePath, moduleDependencies) => {
```

Comment:
Lets have `filePath` instead of `filenamePath`

Reply:
fixed